scripts/emulations/README.md

```diff
@@ -0,0 +1,23 @@
+# Add Validator
+
+This script will help us in emulating various feature in blockchain programmatically. 
+
+<br />
+
+## Requirements
+[NodeJs (v14.17.0 lts)](https://nodejs.org/en/download/ "NodeJs (v14.17.0 lts)")
```

Comment:
Why do we need this?

Reply:
We need this when we run in our local machine without docker.

Reply:
Take a [look](https://github.com/Cerebellum-Network/nodes-installation-scripts/pull/49#discussion_r655489234).

Reply:
Updated with docker file version